modules/swagger-codegen/src/main/resources/typescript-fetch/modelGeneric.mustache

```diff
@@ -0,0 +1,41 @@
+/**
+ * {{{description}}}
+ * @export
+ * @interface {{classname}}
+ */
+export interface {{classname}} {{#parent}}extends {{{parent}}} {{/parent}}{
+{{#additionalPropertiesType}}
+    [key: string]: {{{additionalPropertiesType}}}{{#hasVars}} | any{{/hasVars}};
+
+{{/additionalPropertiesType}}
+{{#vars}}
+    /**
+     * {{{description}}}
+     * @type {{=<% %>=}}{<%datatype%>}<%={{ }}=%>
+     * @memberof {{classname}}
+     */
+    {{name}}{{^required}}?{{/required}}: {{#isEnum}}{{{datatypeWithEnum}}}{{/isEnum}}{{^isEnum}}{{{datatype}}}{{/isEnum}};
+{{/vars}}
+}{{#hasEnums}}
+
+/**
+ * @export
+ * @namespace {{classname}}
+ */
+export namespace {{classname}} {
```

Comment:
namespace for storage inline enum

Used namespace to enum name pretty and Orderly